src/components/movie-card/movie-card.jsx

```diff
@@ -1,20 +1,25 @@
 import PropTypes from "prop-types";
+import Button from "react-bootstrap/Button";
+import Card from "react-bootstrap/Card";
 
 export const MovieCard = ({ movie, onMovieClick }) => {
-  return (
-    <div
-      onClick={() => {
-        onMovieClick(movie);
-      }}
-      >
-        {movie.Title}
-      </div>
-  ); 
+    return (
+        <Card>
+            <Card.Img variant="top" src="https://cache.pressmailing.net/thumbnail/story_hires/c3822451-8a37-4c19-a495-fd3fc0edeb33/image.jpg" />
```

Comment:
@michaelleoniuk , Here is how you can set the image height. 
```suggestion
            <Card.Img variant="top"  height={300} src="https://cache.pressmailing.net/thumbnail/story_hires/c3822451-8a37-4c19-a495-fd3fc0edeb33/image.jpg" />
```